source/publishing_data/04_configuring_a_source/connectors/salesforce.rst

```diff
@@ -49,4 +49,4 @@ Configuration
      * By default, the box is not checked. Check the box to use the sandbox.
    * * Filtered Fields
      * Fields to retrieve from the object.
-     * Indicate the name of the field in the textbox. Relationships can be followed with the following notation: ``GC_OBO_Referential__r.Name``.
+     * Indicate the name of the field in the textbox. Relationships can be followed with the following notation: ``GC_OBO_Referential__r.Name``. Indicate the name of a objects to extract all its fields. Left the area empty to extract all the fields of the targeted object.
```

Comment:
```suggestion
     * Indicate the name of the field in the textbox. Relationships can be followed with the following notation: ``GC_OBO_Referential__r.Name``. Indicate the name of an objects to extract all its fields. Leave the area empty to extract all the fields of the targeted object.
```
Maybe provide a couple exemples here like 'Opportunity.Owner.Name' and 'Opportunity.Owner'